tools/skylark/pathutils.bzl

```diff
@@ -51,7 +51,7 @@ def __remove_prefix(path, prefix):
             # remaining components (len(prefix) - n - 1; the 1 is the current
             # prefix component) less one (since we need to keep at least one
             # component of the path).
-            k = len(path) - (len(prefix) - n - 2)
+            k = len(path) - (len(prefix) - n - 1)
 
```

Comment:
For the `_remove_prefix` function, given `path=
"bazel-bin/cyber/mainboard/mainboard"`, and `prefix=
"bazel-bin/**/mainboard"`, we would expect that
`_remove_prefix(path, prefix)` returns "mainboard".
However, as the original calculation of `k` is wrong, `_remove_prefix(path, prefix)` in this case wouldn't work properly.